pr_testing/run-pr-comparisons

```diff
@@ -387,7 +387,7 @@ if [ "X$RUN_DEFAULT_COMP" = Xtrue ]; then
 fi
 echo "Jobs: $(jobs -p | wc -l)"
 jobs
-
+wait
```

Comment:
why `wait` ? The input files you need are just the `maxmem_profile_*.txt` files ... right? So those should have been download after https://github.com/cms-sw/cms-bot/blob/master/pr_testing/run-pr-comparisons#L151 is run

Reply:
Not all of the workflows were being added to the summary table. I thought it was because the previous jobs weren't complete.

Reply:
For release, we have 45 maxmem profile files [a] and for PR relvals we have 90 files ( due to extra copy) [b].   So ideally all workflows should have been added . Maybe some logic in summary script which drops workflows?

[a]
```
[muzaffar@lxplus978 matrix-results]$ pwd
/cvmfs/cms-ci.cern.ch/jenkins-artifacts/ib-baseline-tests/CMSSW_14_2_X_2024-09-23-1100/el8_amd64_gcc12/-GenuineIntel/matrix-results
[muzaffar@lxplus978 matrix-results]$ ls */maxmem_profile*.txt |wc
     45      45    2371
```

[b]
```
[cmsbuild@cmssdt9-02 runTheMatrix-results]$ pwd
/data/sdt/SDT/jenkins-artifacts/pull-request-integration/PR-ffff10/41711/runTheMatrix-results
[cmsbuild@cmssdt9-02 runTheMatrix-results]$ ls */maxmem_profile_*.txt |wc
     90      90    5588
```

Reply:
The copy with the workflow name added doubled the count.